.github/azure-pipelines.yaml

```diff
@@ -0,0 +1,54 @@
+trigger:
+  branches:
+    include:
+    - master
+  tags:
+    include:
+    - ^v\d+\.\d+(\.\d+)?(-\S*)?$
+
+pr:
+- master
+
+resources:
+  repositories:
+  - repository: azure_template
+    type: github
+    name: mdolab/.github
+    endpoint: mdolab
+
+stages:
+- stage: Test
+  dependsOn: []
+  displayName: Test
+  jobs:
+  - job:
+    pool:
+      vmImage: 'ubuntu-20.04'
+    strategy:
+      matrix:
+        'python36':
+          PYTHON_VERSION: '3.6'
+        'python37':
+          PYTHON_VERSION: '3.7'
+        'python38':
+          PYTHON_VERSION: '3.8'
+    steps:
+    - script: sudo apt-get install texlive-pictures texlive-latex-extra -y
+      displayName: Prepare Repository
+    - script: |
+        python3 -m venv test
+        source test/bin/activate
+        pip install wheel
```

Comment:
Why is `wheel` needed here?

Reply:
`wheel` is needed because the installation for `testflo` fails without it. The virtual environment (and the provided python environment) come without it.

Reply:
With Python correctly set up I presume `wheel` will be installed by default. Would you mind checking?